tests/RespTheme/desktopBackgroundCest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+use \page\RespTheme\Customizer;
+use \page\RespTheme\LogInAndLogOut;
+
+class desktopBackgroundCest
+{
+    public function _before(RespThemeTester $I)
+    {
+    }
+
+    // tests
+    public function tryToTest(RespThemeTester $I,  LogInAndLogOut $loginAndLogout, Customizer $customizer)
```

Comment:
give a descriptive name to the function and Cest file name should start with a capital letter.